CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ##### Enhancements
 
+* TypeBodyLengthRule now does not count comment lines.  
```

Comment:
"comment or whitespace lines"? Same with the one for `FunctionBodyLengthRule` below.